<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Machine_VisionClass</class>
 <widget class="QMainWindow" name="Machine_VisionClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Machine_Vision</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="PbnOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblChosenFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>STATUS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutputImage">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>281</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Gamma">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ThresholdValue">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold Valueï¼š</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ThresholdValue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuPre_processing">
    <property name="title">
     <string>Pre-processing</string>
    </property>
    <addaction name="actionSmoothing"/>
    <addaction name="actionSmoothing_Data"/>
    <addaction name="actionMedian"/>
    <addaction name="actionEqualization"/>
    <addaction name="actionPowerLaw"/>
    <addaction name="actionConservative_Smoothing"/>
    <addaction name="actionSecond_Derivative_Sharpening"/>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>Segmentation</string>
    </property>
    <addaction name="actionThresholding"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPre_processing"/>
   <addaction name="menuSegmentation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSmoothing">
   <property name="text">
    <string>Smoothing_at</string>
   </property>
  </action>
  <action name="actionSmoothing_Data">
   <property name="text">
    <string>Smoothing_Data</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>Median</string>
   </property>
  </action>
  <action name="actionEqualization">
   <property name="text">
    <string>Equalization</string>
   </property>
  </action>
  <action name="actionPowerLaw">
   <property name="text">
    <string>PowerLaw</string>
   </property>
  </action>
  <action name="actionConservative_Smoothing">
   <property name="text">
    <string>Conservative Smoothing</string>
   </property>
  </action>
  <action name="actionSecond_Derivative_Sharpening">
   <property name="text">
    <string>Second Derivative Sharpening</string>
   </property>
  </action>
  <action name="actionThresholding">
   <property name="text">
    <string>Thresholding</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="machine_vision.qrc"/>
 </resources>
 <connections/>
</ui>
